rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -681,6 +687,45 @@ func Qualify_BaseProduct_EquityForward: <"Qualifies a product as having the Asse
         Qualify_AssetClass_Equity(economicTerms) = True
             and economicTerms -> payout -> forwardPayout only exists
 
+func Qualify_EquityForward_PriceReturnBasicPerformance_SingleName: <"Qualifies a product as a plain vanilla Equity Option.  The determination of the qualification is based on the economic terms and the following criteria: 1) An option product  for which the underlier is a single stock and 2) No special option feature exists other than option averaging.">
+    [qualification Product]
+    inputs:
+        economicTerms EconomicTerms (1..1)
+    output:
+        is_product boolean (1..1)
+            [synonym ISDA_Taxonomy_v1 value "EquityForward_PriceReturnBasicPerformance_SingleName"]
+            [synonym ISDA_Taxonomy_v2 value "EquityForward_PriceReturnBasicPerformance_SingleName"]
+    set is_product:
+        Qualify_AssetClass_Equity(economicTerms) = True
+            and economicTerms -> payout -> forwardPayout only exists
+            and economicTerms -> payout -> forwardPayout only-element -> underlier -> security exists
+
+func Qualify_EquityForward_PriceReturnBasicPerformance_SingleIndex: <"Qualifies a product as a plain vanilla Equity Option.  The determination of the qualification is based on the economic terms and the following criteria: 1) An option product  for which the underlier is a single stock and 2) No special option feature exists other than option averaging.">
+    [qualification Product]
+    inputs:
+        economicTerms EconomicTerms (1..1)
+    output:
+        is_product boolean (1..1)
+            [synonym ISDA_Taxonomy_v1 value "EquityForward_PriceReturnBasicPerformance_SingleIndex"]
+            [synonym ISDA_Taxonomy_v2 value "EquityForward_PriceReturnBasicPerformance_SingleIndex"]
+    set is_product:
+        Qualify_AssetClass_Equity(economicTerms) = True
+            and economicTerms -> payout -> forwardPayout only exists
+            and economicTerms -> payout -> forwardPayout only-element -> underlier -> index exists
+
+func Qualify_EquityForward_PriceReturnBasicPerformance_Basket: <"Qualifies a product as a plain vanilla Equity Option.  The determination of the qualification is based on the economic terms and the following criteria: 1) An option product  for which the underlier is a single stock and 2) No special option feature exists other than option averaging.">
```

Comment:
And again on this one, we're looking for a basket so we just need to update the function comment to read "basket" rather than "single stock" 🙂